simple_scripts/faraday_program.py

```diff
@@ -0,0 +1,85 @@
+from subprocess import call
+import os
+import time
+import sys
+
+#Detect operating system
+#LINUX: Assumed terminal is gnome-terminal standard with Ubuntu
+running_os = sys.platform
+
+# Proxy
+proxy_callsign = 'REPLACEME'
+proxy_nodeid = 0
```

Comment:
Yeah, TravisCI fails otherwise. Adding a comment "Replace with proxy_nodeid of local unit to be programmed"